src/java/org/apache/cassandra/db/filter/SliceQueryFilter.java

```diff
@@ -466,7 +470,7 @@ public boolean hitTombstoneWarnThreshold() {
 
     public int lastTombstones()
     {
-        return columnCounter == null ? 0 : columnCounter.tombstones();
+        return columnCounter == null ? 0 : columnCounter.tombstones() + rangeTombstoneCounter.getCount();
```

Comment:
I would prefer to separate this into its own variable / metric, because it allows us to:
1) clearly see if there are errors in our methodology
2) easily differentiate between range and non-range tombstones in analysis
3) compare future tombstone metrics to past

Reply:
👍